components/dashboard/src/teams/TeamUsageBasedBilling.tsx

```diff
@@ -158,11 +159,13 @@ export default function TeamUsageBasedBilling() {
         }
     };
 
+    const attributionId: AttributionId = { kind: "team", teamId: team.id };
```

Comment:
Given the above, maybe it's okay to have a "garbage" `attributionId` until both `team` (and `subscriptionId`, but that's less relevant here) are loaded:

```suggestion
    const attributionId: AttributionId = { kind: "team", teamId: team?.id || "" };
```

Reply:
Alternatively, you could make `attributionId` of type `AttributionId?`, and leave it `undefined` until the `team` is loaded.

Reply:
Thanks. Applied your suggestion.